src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,6 @@
+body {
```

Comment:
always set font to the `html {...}` tag

Reply:
Done. Why is it better to set font to html? As all element are in body tag anyway